CHANGELOG.md

```diff
@@ -35,6 +36,41 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.11.0]
+
+> Release date: 2022/02/17
+
+### Added
+
+- decK now populates core entities defaults from their schema leveraging the Admin API.
+  [#573](https://github.com/Kong/deck/pull/573)
+- decK now populates plugins defaults from their schema leveraging the Admin API.
+  [#562](https://github.com/Kong/deck/pull/562)
+- decK prevents users to interact with any internal-only Konnect plugins.
+  [#564](https://github.com/Kong/deck/pull/564)
+- decK now supports mTLS Kong Admin API authentication via
+  `--tls-client-cert`/`--tls-client-cert-file` and
+  `--tls-client-key`/`--tls-client-key-file` flags.
+  [#509](https://github.com/Kong/deck/pull/509)
+- decK `validate` now has a `--online` flag enabling entities validation against
+  the Admin API. This lets users validate their configuration before deploying it.
+  [#502](https://github.com/Kong/deck/pull/502)
+
+### Fixes
+
+- decK now can set zero values in entity fields.
+  [#580](https://github.com/Kong/deck/pull/580)
+- fix unexpected behaviour when running `sync` against multiple workspaces at once.
```

Comment:
```suggestion
- Attempting to run `sync` on multiple workspaces now returns an error rather th
  an applying a single workspace to all entities.
```

Reply:
^ typo/accidental line break mid-word, otherwise the suggestion looks good.